src/util/File.h

```diff
@@ -200,22 +199,16 @@ class File {
   //! Read nofBytesToRead bytes from file starting at the given offset.
   //! Returns the number of bytes read or the error returned by pread()
   //! which is < 0
-  ssize_t read(void* targetBuffer, size_t nofBytesToRead, off_t offset,
-               ad_utility::SharedConcurrentTimeoutTimer timer = nullptr) {
```

Comment:
This was always nullptr, and setting this to an actual value alters the batching behaviour so I remove it entirely.